lib/thredded.rb

```diff
@@ -65,11 +66,21 @@ module Thredded
   self.user_name_column = :name
   self.content_visible_while_pending_moderation = true
   self.show_topic_followers = false
+  self.messageboards_order = :position
 
   def self.user_display_name_method
     @@user_display_name_method || user_name_column
   end
 
+  def self.messageboards_order=(value)
```

Comment:
Some simple docs so that folks get warnings in their IDEs if they pass an invalid value:

```ruby
# @param [:position, :topics_count_desc, :last_post_at_desc] value
```